src/test-support/FakeKuma.ts

```diff
@@ -39,8 +39,8 @@ export class KumaModule {
       [
         'internal',
         'external',
-        // 'gateway_delegated',
-        // 'gateway_builtin',
+        'gateway_delegated',
+        'gateway_builtin',
```

Comment:
I'm trying to remember why I commented these out originally (i.e. I didn't just delete as I was unsure). I think I wasn't sure if there were places where you would guarantee that you wouldn't get the gateway types, in which case we might need to add an arg to specify what you want.

Again bit of a nit so not a blocker here, unless it affects other tests of course, maybe we just need to double check where we already use this, maybe you've done that already, lmk

Reply:
It seemed to work fine with our current tests so I think it’s better to put them in. Maybe it exposes similar issues earlier.

Reply:
Ok yeah no worries, I think I half-remembered, maybe there is something we extra we should do/could do here, but we can do it later and I don't think its super important anyway, more of a dev nicety, will resolve.